create-electron-documentation/templates/documentation.md

```diff
@@ -0,0 +1,29 @@
+---
+title: ${title}
+description: ${description}
+slug: ${slug}
+hide_title: true
+---
+
+# ${title}
+
+## Overview
+
+<!-- ✍ Update this section if you want to provide more details -->
+
+${description}
+
+### Examples
+
+<!--
+    ✍ Feel free to rename this section,
+    add new ones, etc.
+-->
+
+```fiddle docs/fiddles/${slug}
```

Comment:
Might be worth adding a comment above this line explaining the `fiddle docs/fiddles/${slug}` part (or at least linking to something else that explains it). I can see users with previous markdown experience going straight for `js` or `ts`

Reply:
Good idea!

Reply:
@Ethan-Arrowood @erickzhao what do you think of the following?

```suggestion
<!--
    Because Electron examples usually require multiple files (HTML, CSS, JS
    for the main and renderer process, etc.), we use this custom code block
    for Fiddle (https://www.electronjs.org/fiddle).
    Please modify any of the files in the referenced folder to fit your
    example.
    The content in this codeblock will not be rendered in the website so you
    can leave it empty.
-->
```fiddle docs/fiddles/${slug}
```

Reply:
Sounds good. A good limitation to note is that the "Open in Fiddle" link only works once the doc example is merged in `e/e`.

Reply:
There's no way for them to test it locally?
We might have to look at improving Fiddle's documentation for exporting/importing to make this easier 🤔

Reply:
> There's no way for them to test it locally?

For specifically the `fiddle.electronjs.org` redirect, no.

The link works by sending the URL of the folder as a param into `https://fiddle.electronjs.org`. I don't think there's a way to make that service work with local files.

However, we could replicate this feature locally by somehow allowing the fiddle protocol launch to work with a local folder, but I'm not sure how that would look technically.